ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master.yaml

```diff
@@ -66,6 +66,10 @@ tag_specification:
   name: "4.8"
   namespace: ocp
 tests:
+- as: e2e-openstack
+  steps:
+    cluster_profile: openstack-vexxhost
```

Comment:
Here you set the Vexxhost cluster profile, however `openshift-e2e-openstack-minimal` contains the flavor name and the external network name of MOC

Reply:
https://github.com/openshift/release/blob/9c49e8e6afb3ef7f219dea8e38c6a200a7e00896/ci-operator/step-registry/openshift/e2e/openstack/minimal/openshift-e2e-openstack-minimal-workflow.yaml#L9-L11